### PR TITLE
Fix Windows Python 3 #!pipe

### PR DESCRIPTION
**Pull request by kazarmy**
**Detailed description**

This pr makes the following changes/fixes to the Python 3 Windows #!pipe code:

1. The R2PIPE_PATH env var already contains the full path to the named pipe, so a prefix shouldn't be added to it.
1. Need to use the W versions of the Win32 api for Python 3 strings.
1. Need to encode the r2 command in utf-8 first before sending it down the pipe to r2.
1. Removed the `e scr.color=false` invocation for consistency with non-Windows code.
1. Better error messaging.

~~Unfortunately, this pr is necessary but not sufficient for correct #!pipe operation on Windows, but can this pr be merged in first and the version bumped?~~

This is the companion pr to radareorg/radare2#17139.

**Test plan**

See radareorg/radare2#17139 for a test script.

**Closing issues**

...

### CODE REVIEW

**Comment by kazarmy** [comment]
The companion pr on the r2 side is radareorg/radare2#17139.

**Comment by kazarmy** [comment]
Updated the description.

**Comment by kazarmy** [comment]
Sorry, "r2pipe" was too broad ... "#!pipe" is more accurate

File: python/r2pipe/open_base.py [hunk 1]
```diff
@@ -117,8 +116,8 @@ def __init__(self, filename="", flags=[]):
             if os.name == "nt":
                 mypipename = os.environ["r2pipe_path"]
                 while 1:
-                    hPipe = windll.kernel32.CreateFileA(
-                        szPipename + mypipename,
+                    hPipe = windll.kernel32.CreateFileW(
+                        mypipename,
```
[the remark this answers is not in the document]
Comment (XVilka) [reply python/r2pipe/open_base.py:120]: No, we don't.

[the remark this answers is not in the document]
Comment (a1ext) [reply python/r2pipe/open_base.py:120]: ok then

**Pull request merged**